decidim-core/spec/system/user_activity_spec.rb

```diff
@@ -40,6 +40,21 @@ def resource_types
     create(:dummy_resource, component: component, published_at: Time.current)
   end
 
+  let!(:resource_types) do
```

Comment:
is this being used?

Reply:
I changed it so now uses a specific array with the resource_types requested. The spec uses that variable